src/view/seed-mobile.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import MainContent from '../component/main-content';
+import { Text } from '../component/text';
+import { Background } from '../component/background';
+import { Header, Title } from '../component/header';
+import { Button, BackButton, GlasButton } from '../component/button';
+import LightningBoltIcon from '../asset/icon/lightning-bolt';
+import { color, font } from '../component/style';
+
+//
+// Seed (Mobile) View
+//
+
+const styles = StyleSheet.create({
+  header: {
+    height: 150,
+  },
+  title: {
+    width: 200,
+  },
+  background: {
+    flex: 1,
+    justifyContent: 'space-between',
+    backgroundColor: color.blackDark,
+  },
+  seed: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  pagination: {
+    paddingBottom: 10,
+  },
+});
+
+const SeedView = ({ store, auth }) => (
+  <Background image="purple-gradient-bg">
+    <Header separator style={styles.header}>
+      <BackButton onPress={() => auth.initPrevSeedPage()} />
+      <Title
+        style={styles.title}
+        keepCase
+        title="Write down each word and store it in a safe place."
+      >
```

Comment:
Title isn't semantically correct here. We should use one of the Text of Copy components.

Reply:
fixed